metric/doc.go

```diff
@@ -68,6 +68,37 @@ asynchronous measurement, a Gauge ([Int64ObservableGauge],
 See the [OpenTelemetry documentation] for more information about instruments
 and their intended use.
 
+# Measurements
+
+Measurements are made by recording values and information about the values with
+an instrument. How these measurements are recorded depends on the instrument.
+
+Measurements for synchronous instruments ([Int64Counter], [Int64UpDownCounter],
+[Int64Histogram], [Float64Counter], [Float64UpDownCounter], [Float64Histogram])
+are made using the instrument methods directly. All counter instruments have an
+`Add` method that is used to measure an increment value, and all histogram
+instruments have a `Record` method to measure a data point.
+
+Asynchronous instruments ([Int64ObservableCounter],
+[Int64ObservableUpDownCounter], [Int64ObservableGauge],
+[Float64ObservableCounter], [Float64ObservableUpDownCounter],
+[Float64ObservableGauge]) make measurements within a callback function. The
```

Comment:
For the synchronous instruments, we said passively that "measurements are made." Do the instruments "make" these measurements, or do they allow us callers to make measurements using the instruments?
```suggestion
and [Float64ObservableGauge]) accept measurements within a callback function. The
```

Reply:
Yeah, maybe we can tweak the wording here.

I don't think is accept is correct though. The instruments themselves don't accept the measurements.

Maybe say they are used to make measurements?

Reply:
I think "record measurements" is the correct verb

Reply:
Updated, PTAL